src/erlcloud_kinesis.erl

```diff
@@ -406,6 +407,57 @@ put_record(StreamName, PartitionKey, Data, ExplicitHashKey, Ordering, Config) wh
   Json = [{<<"StreamName">>, StreamName}, {<<"PartitionKey">>, PartitionKey}, {<<"Data">>, base64:encode(Data)}, {<<"ExplicitHashKey">>, ExplicitHashKey}, {<<"SequenceNumberForOrdering">>, Ordering}],
   erlcloud_kinesis_impl:request(Config, "Kinesis_20131202.PutRecord", Json).
 
+%%------------------------------------------------------------------------------
+%% Kinesis API:
+%% [https://docs.aws.amazon.com/kinesis/latest/APIReference/API_PutRecords.html]
+%%
+%% ===Example===
+%%
+%% This operation puts a data record into an Amazon Kinesis stream from a producer.
+%%
+%% `
+%% erlcloud_kinesis:put_records(<<"test">>, [{<<"asdasd">>, <<"asdasd">>, <<"asdasd">>}]).
```

Comment:
This doesn't seem like an acceptable example. Please follow the style of the existing examples in this module.